### ISSUE TITLE
Activity Log: backup complete item should not be rewound

### ISSUE THREAD

**Issue by keoshi** [Automattic/wp-calypso#22283]
When confirming a site restore the confirm dialog appears above the selected item, letting me know the site will be rewound to this point and all other items above will be discarded.

![image](https://user-images.githubusercontent.com/390760/36024013-4fc01e4a-0d86-11e8-9c54-2eec62fc06f6.png)

However, once the restore is complete, the initial “full backup” item is also marked as discarded:

![image](https://user-images.githubusercontent.com/390760/36024047-67e7a100-0d86-11e8-85b3-dafd5b5abf86.png)

EDIT: these two screenshots are not part of the same flow, just illustrating styling differences before and after a restore.

@eliorivero can you take a look, please?

**Comment by eliorivero** [comment]
Thanks for the report. Ok, we'll look into keeping the backup used for the rewind un-dismissed, as well as the the rewind success event.

**Comment by dmsnell** [comment]
@keoshi is there any way you can share the site on which that was viewed? I would like to check the events themselves.

**Comment by eliorivero** [comment]
Provided required info to Dennis on Slack.

**Comment by dmsnell** [comment]
@keoshi is it possible to try again with that same event and see if the behavior remains? I'm asking because there were recent changes the may address this and I want to rule out the easy things if we can.

if you can't reproduce on this site then we will have to try and find ways to make this reproducible.

**Comment by dmsnell** [comment]
this is all too confusing.

I'm going to stop for the day but I suspect that what's happening is related to the way we butcher timestamps. the `rewind_id` is usually a string version of a timestamp with fractional seconds and I have seen precision down to ten-thousandths of a second. that is `ss.SSSS` if that makes sense.

therefore since we truncate the timestamp when calculating `is_discarded` to the ms then we may simply be ending with a 9-out-of-10 chance that the backup will appear discarded when it's not.

**Comment by dmsnell** [comment]
Hopefully related and hopefully resolving the issue: D10486

**Comment by eliorivero** [comment]
With discarded events of the equation, we can now close this.
